src/zoid/buttons/component.jsx

```diff
@@ -1266,13 +1272,23 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
 
       supportedNativeBrowser: {
         type: "boolean",
-        value: isSupportedNativeBrowser,
+        value: ({ props }) => {
+          if (props?.fundingSource === FUNDING.VENMO) {
+            return isSupportedNativeVenmoBrowser(props.experiment);
+          }
+          return isSupportedNativeBrowser();
+        },
         queryParam: true,
       },
 
       supportsPopups: {
         type: "boolean",
-        value: () => userAgentSupportsPopups(),
+        value: ({ props }) => {
+          if (props?.fundingSource === FUNDING.VENMO) {
```

Comment:
This method will be called during button render. I am bit concerned if the fundingSource will be available during render?